definitions/contested/json/CaseTypeTab/CaseTypeTab.json

```diff
@@ -4393,7 +4674,7 @@
     "TabDisplayOrder": 15,
     "CaseFieldID": "nottinghamCourtList",
     "TabFieldDisplayOrder": 15,
-    "FieldShowCondition": "regionList=\"midlands\" AND midlandsFRCList=\"nottingham\""
+    "FieldShowCondition": "hearingDate !=\"\" AND hearing_regionList=\"\" AND regionList=\"midlands\" AND midlandsFRCList=\"nottingham\""
```

Comment:
Is this condition correct? 
If regionList =="" and hearing_regionList="midlands" for new cases this condition will fail.

similarly others.

Reply:
I think so, this field is not supposed to show on new cases